tests/unit/test_token.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import urllib3
+from xrdsst.controllers.token import TokenController
+
+class TestInit(unittest.TestCase):
```

Comment:
TestInit is the test class created for testing the InitController, please use another name here, e.g. TestToken